com.unity.shadergraph/Editor/Data/Nodes/Math/Basic/MultiplyNode.cs

```diff
@@ -36,11 +36,9 @@ enum MultiplyType
 
         public override bool hasPreview => true;
 
-        string GetFunctionHeader()
+        string GetFunctionName()
         {
-            return "Unity_Multiply" + "_" + concretePrecision.ToShaderString()
-                + (this.GetSlots<DynamicVectorMaterialSlot>().Select(s => NodeUtils.GetSlotDimension(s.concreteValueType)).FirstOrDefault() ?? "")
-                + (this.GetSlots<DynamicMatrixMaterialSlot>().Select(s => NodeUtils.GetSlotDimension(s.concreteValueType)).FirstOrDefault() ?? "");
+            return $"Unity_Multiply_{FindSlot<MaterialSlot>(Input1SlotId).concreteValueType.ToShaderString()}_{FindSlot<MaterialSlot>(Input2SlotId).concreteValueType.ToShaderString()}";
```

Comment:
Dunno why we had function header (actual function name in code) defined differently than function name (only used for deduplication)..  but it is a requirement that they match now.  Kept the types generic so they work properly.